packages/commonwealth/client/scripts/views/components/user_survey_popup.tsx

```diff
@@ -0,0 +1,156 @@
+/* @jsx m */
+
+import m from 'mithril';
+import app from 'state';
+
+import 'components/user_survey_popup.scss';
+
+import { CWButton } from './component_kit/cw_button';
+import { CWCheckbox } from './component_kit/cw_checkbox';
+import { CWGrowl } from './component_kit/cw_growl';
+import { CWText } from './component_kit/cw_text';
+import { CWIcon } from './component_kit/cw_icons/cw_icon';
+
+const USER_SURVEY_DISPLAY_INTERVAL = 1000 * 60 * 1; // How long to wait before redisplaying the survey
+
+type UserSurveyViewAttrs = {
+  disabled: boolean;
+  checked: boolean;
+  onRedirectClick: () => void;
+  onClose: () => void;
+  onCheckboxClick: () => void;
+};
+
+class UserSurveyView implements m.ClassComponent<UserSurveyViewAttrs> {
+  view(vnode) {
+    const { disabled, checked, onRedirectClick, onClose, onCheckboxClick } =
+      vnode.attrs;
+    return (
+      <CWGrowl position="bottom-right" disabled={disabled}>
+        <div class="UserSurveyPopup">
```

Comment:
why is this div necessary? isn't the CWGrowl wrapper enough?

Reply:
Thats probably true, I will refactor.

Reply:
hmm actually, since the children of a growl are just displayed inside a card, isn't it okay to have styling like this? there are other places in the codebase where a card wraps a div that customizes the styles inside the card.